cookbooks/bcpc-hadoop/metadata.rb

```diff
@@ -7,6 +7,7 @@
 version          "0.1.0"
 
 depends "bcpc", ">= 0.5.0"
+depends "bach_sqoop"
```

Comment:
I suppose the dependency should be inverted: 
- `bach_sqoop` should depend on `bcpc-hadoop` instead
- Instead of including `bach_sqoop` in `bcpc-hadoop::datanode`, we should add `bach_sqoop` to `[BCPC-Hadoop-Worker role](https://github.com/bloomberg/chef-bach/blob/master/roles/BCPC-Hadoop-Worker.json#L19)`